static/js/components/LearnerSearch.js

```diff
@@ -10,13 +11,26 @@ import {
 } from 'searchkit';
 import Grid, { Cell } from 'react-mdl/lib/Grid';
 import Card from 'react-mdl/lib/Card/Card';
+import iso3166 from 'iso-3166-2';
 
 import LearnerResult from './search/LearnerResult';
 import CountryRefinementOption from './search/CountryRefinementOption';
 import FilterVisibilityToggle from './search/FilterVisibilityToggle';
 import HitsCount from './search/HitsCount';
 import type { Option } from '../flow/generalTypes';
 
+let makeTranslations: () => Object = () => {
+  let translations = {};
+  for (let code of Object.keys(iso3166.data)) {
+    translations[code] = iso3166.data[code].name;
+    for (let stateCode of Object.keys(iso3166.data[code].sub)) {
+      translations[stateCode] = iso3166.data[code].sub[stateCode].name;
+    }
+  }
+
+  return translations;
```

Comment:
this object has about 5200 keys when all is said and done. probably not a _terribly_ significant memory/performance hit, but i'm wondering if there's an easy-enough way to build this object on demand and/or cache the values that we've already seen